plugins/modules/vcenter_standard_key_provider.py

```diff
@@ -0,0 +1,670 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2021, Ansible Project
+# Copyright: (c) 2021, VMware, Inc. All Rights Reserved
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+---
+module: vcenter_standard_key_provider
+short_description: Add, reconfigure or remove Standard Key Provider on vCenter server
+description: >
+  This module is used for adding, reconfiguring or removing Standard Key Provider on vCenter server.
+  Refer to VMware docs for more information: L(Standard Key Provider,
+  https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.security.doc/GUID-6DB1E745-9624-43EA-847C-DD2F767CB94B.html)
+author:
+  - Diane Wang (@Tomorrow9) <dianew@vmware.com>
+requirements:
+  - python >= 2.6
```

Comment:
```suggestion
  - python >= 3.6
```

Please test against Python 3.6 and onwards.

Reply:
Sure, thanks.